### PR TITLE
improvements 4

### PR DESCRIPTION
**Pull request by bladicito**

### CODE REVIEW

File: components/modal/Modal.tsx [hunk 1]
```diff
@@ -48,9 +48,11 @@ const Modal = () => {
       className={`flex items-center justify-center fixed  rounded  ${positionCommon} ${transitionOptions} ${isOpen ? 'opacity-100' : 'opacity-0 pointer-events-none'}`}
     >
       <div
-        className={`modal-backdrop fixed ${positionCommon} bg-violet-500 bg-opacity-50`}
+        className={`modal-backdrop fixed ${positionCommon} bg-violet-500 bg-opacity-50  ${isOpen ? 'block' : 'hidden'}`}
       />
-      <div className="modal-wrapper w-[calc(100%-48px)] h-[calc(100%-48px)] px-[50px] py-[50px] z-40">
+      <div
+        className={`modal-wrapper w-[calc(100%-48px)] h-[calc(100%-48px)] px-[50px] py-[50px] z-40 ${isOpen ? 'block' : 'hidden'}`}
```
Comment (lehmi11) [inline components/modal/Modal.tsx:54]: is `modal-wrapper `an identifier? if yes, do we need it?

File: components/post-image/PostImage.tsx [hunk 2]
```diff
@@ -18,19 +18,18 @@ interface IProps {
...
-  const imageProportions = 'pb-[calc((8/17)*100%)]';
-  const imageTransition = 'transition-all ease-in delay-500 duration-500';
+  const imageTransition = 'transition-opacity duration-500 delay-500';
 
   const { dispatchModal } = useModal();
   return (
     <div className="relative">
       <div
-        className={`${loaded ? '-z-10 opacity-0' : 'z-10 opacity-100'} absolute top-0 left-0 right-0 bottom-0 ${imageProportions} ${imageTransition}`}
+        className={`${loaded ? '-z-10 opacity-0' : 'z-10 opacity-100'} absolute top-0 left-0 right-0 bottom-0 mumble-image ${imageTransition}`}
```
Comment (lehmi11) [inline components/post-image/PostImage.tsx:27]: is mumble-image an identifier or a global css class?

**Review by lehmi11 — APPROVED** [review]
Nice improvements, especially the loading components

File: components/modal/Modal.tsx (hunk 1 above)
Comment (bladicito) [reply components/modal/Modal.tsx:54]: I find the lack of component names disturbing for deployment, testing, bugFixing. The "Smartive way" avoids to have them but, I dont want to work for smartive. I want to work only smart. 
We could remove them as per final task. Even so, improving css later on, can be really pain in the while inspecting the markup on the developer tools.

File: components/post-image/PostImage.tsx (hunk 2 above)
Comment (bladicito) [reply components/post-image/PostImage.tsx:27]: mumble image is a global class.

File: components/modal/Modal.tsx (hunk 1 above)
Comment (lehmi11) [reply components/modal/Modal.tsx:54]: I only wanted to know what purpose this string has. I think it is a good idea to keep it for debugging purposes. Don't think we should go through the code and remove those classes.

**Pull request merged**